userguide/one_shot_object_detection/README.md

```diff
@@ -0,0 +1,109 @@
+# One-Shot Object Detection
+
+One-Shot object detection is the task of learning to detect a set of categories from a handful of images. Unlike the Object Detector which requires many varied examples of objects in the real world, the One-Shot detector only requires a single canonical example of an object to train the classifier on and performs predictions for that category in the wild.
+
+Given an image and a one-shot detector trained on category 'stop sign', the output prediction will look like:
+
+![Stop Sign vector image](images/red_stop_sign.png) ![Stop Sign prediction image](images/stop_sign_prediction_sample.png)
+
+
+#### Introductory Example
+
+In this example, the goal is to localize where in the image instances of the category 'stop sign' are present. For this task we supply a single image of a stop sign as a starter image:
+
+
+```
+import turicreate as tc
+
+# Load the starter images
+starter_images = tc.SFrame({'image':[tc.Image('stop_sign.png')],
+                   'label':['stop_sign']})
+
+# Load test images
+test_data = tc.SFrame({'image':[tc.Image('stop_sign_photo1.png'), 
+                                tc.Image('stop_sign_photo2.png')],
+                        'label':['stop_sign', 'stop_sign']})
+
+# Create a model                                      
+model = tc.one_shot_object_detector.create(starter_images, 'label')
```

Comment:
Since this step can take quite a while, I'd probably add a comment giving the user some expectation of how long this will take.

Reply:
The time would be a function of number of categories, number of augmented images used, and number of iterations. How do you think I should frame that? For example, for the one stop sign class, 951 augmentation images, and the default 20k iterations took between 8-12 hours.

Reply:
I would probably just add a comment saying this will take several hours.